opendc-model-odc/jpa/src/main/resources/jpa/schema.xml

```diff
@@ -191,6 +191,18 @@
                     <cascade-persist/>
                 </cascade>
             </many-to-many>
+            <many-to-many name="dependents" target-entity="Task">
+                <join-table name="task_dependencies">
+                    <join-column name="first_task_id"/>
+                    <inverse-join-column name="second_task_id"/>
+                </join-table>
+                <cascade>
```

Comment:
@fabianishere Do you mean this block? Because after removing it I still encounter the error.

Reply:
Also tried removing the one in `depedencies`, still the same error.

Reply:
I am having issues reproducing the error. Could you post the command that you run?

Reply:
See #39 for the fix